x-pack/test/functional/apps/ml/anomaly_detection/fixtures/lens_to_ml.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
When we first talked about the location for the new kbnArchiver fixture, I've suggested to put it next to the tests. But now I found that there's already a common location for those in `x-pack/test/functional/fixtures/kbn_archiver`. I'd suggest to move this fixture into a `ml` sub-directory of this common directory, so we'd use it similarly to our esArchives in  `x-pack/test/functional/es_archives`

Reply:
https://github.com/elastic/kibana/pull/133899/commits/9b5de912e6751d39a725e4cc348f6ea9bf7ba7e7